website/docs/guides/dependencies.md

```diff
@@ -6,56 +6,62 @@ To add a dependency, simply declare it in your `Scarb.toml`.
 > [!WARNING]
 > Using Git repositories as a foundation for package management is not the recommended approach anymore.
 > Instead, [registries](../registries/overview.md) are now the primary way to manage dependencies.
-> This guide will be updated to reflect the new approach after official registry is out of beta and considered stable.
-> For details on how to specify dependencies from the official registry, see [here](../reference/specifying-dependencies#specifying-dependencies-from-official-registry).
+> For details on how to specify dependencies from the official registry,
+> see [here](../reference/specifying-dependencies#specifying-dependencies-from-official-registry).
+> If you want to rely on git dependencies instead of the recommended way, you can learn more
+> [here](../reference/specifying-dependencies#specifying-dependencies-from-git-repositories).
 
 ## Adding a dependency
 
-If your `Scarb.toml` doesn't already have a `[dependencies]` section, add it, then list the package name and the URL to
-its Git repository.
-This example adds a dependency on the [`alexandria`](https://github.com/keep-starknet-strange/alexandria) package (note
-that Alexandria is a collection of multiple packages, and we will use `alexandria_math` as an example in this guide):
+If your `Scarb.toml` doesn't already have a `[dependencies]` section, add it, then list the package name and the version
+required. You can search for packages to use through the [scarbs.xyz registry website](https://scarbs.xyz/).
+This example adds a dependency on the [`openzeppelin_merkle_tree`](https://github.com/OpenZeppelin/cairo-contracts)
+package (note that OpenZeppelin is a collection of multiple packages, and we will use only one of them as an example in
+this guide). To see all available versions of some package, you can see
+its [registry page](https://scarbs.xyz/packages/openzeppelin_merkle_tree). At the time of writing this guide, the latest
+version of the `openzeppelin_merkle_tree` package is `0.17.0`, which is the version we will use.
 
 ```toml
 [dependencies]
-alexandria_math = { git = "https://github.com/keep-starknet-strange/alexandria.git" }
+openzeppelin_merkle_tree = "0.17.0"
 ```
 
-You can pin a Git dependency to concrete commit, branch or a tag using one of the following extra fields that can be
-passed along `git`: `branch`, `tag` and `rev`.
+Using `"0.17.0"` as version requirement means, that you want to use a version `0.17.0` or newer, but not `0.18.0` or
+newer. To accept only a specific version, you can use `"=0.17.0"`. You can learn more about specifying version
```

Comment:
nitpick: I think phrasing here could be better. Maybe something like "... that you want to use a version `0.17.0` or newer, up until `0.18.0`" or something similar?

Reply:
Yes, thanks!